package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "noffle",
+  "bugs": {},
+  "gx": {
+    "dvcsimport": "github.com/noffle/ipfs-embedded-shell"
```

Comment:
probably want to change the path here to point to my repo (otherwise deps rewriting won't work properly)